static/js/ui.js

```diff
@@ -123,13 +123,21 @@ exports.show_failed_message_success = function (message_id) {
     });
 };
 
+exports.get_hotkey_deprecation_notice = function (originalHotkey, replacementHotkey) {
+    return i18n.t(
+        'We\'ve replaced the "'
+            + originalHotkey + '" hotkey with "'
+            + replacementHotkey + '" to make this common shortcut easier to trigger.'
+    );
```

Comment:
You need to do the templating differently to make i18n work correctly (basically, passing in the arguments to `i18n.t`) for languages with a different word order than English.

See http://zulip.readthedocs.io/en/latest/translating/ for details.